### PR TITLE
snakeYaml upgrade

### PR DESCRIPTION
**Pull request by ljacomet**

### CODE REVIEW

**Comment by ljacomet** [comment]
@bot-gradle test RFM

**Comment by ljacomet** [comment]
@bot-gradle test QF

**Comment by JLLeitschuh** [comment]
@bot-gradle test QF

**Comment by JLLeitschuh** [comment]
@bot-gradle test QF

**Comment by big-guy** [comment]
@bot-gradle RfM

**Comment by big-guy** [comment]
@bot-gradle RFM

**Comment by big-guy** [comment]
@bot-gradle test RFM

File: subprojects/platform-native/src/main/java/org/gradle/nativeplatform/toolchain/internal/swift/SwiftDepsHandler.java [hunk 1]
```diff
@@ -83,8 +84,14 @@ private void write(File moduleSwiftDeps, final SwiftDeps swiftDeps) {
...
-                Yaml yaml = new Yaml();
-                yaml.dump(swiftDeps, bufferedWriter);
+                PrintWriter pw = new PrintWriter(bufferedWriter);
+                pw.println("version: \"" + swiftDeps.version + "\"");
+                pw.println("options: \"" + swiftDeps.options + "\"");
+                pw.println("build_time: [" + swiftDeps.build_time.get(0) + ", " + swiftDeps.build_time.get(1) + "]");
+                pw.println("inputs:");
+                swiftDeps.inputs.forEach((file, timestamp) -> {
+                    pw.println("  \"" + file + "\": [" + timestamp.get(0) + ", " + timestamp.get(1) + "]");
+                });
```
Comment (donat) [inline subprojects/platform-native/src/main/java/org/gradle/nativeplatform/toolchain/internal/swift/SwiftDepsHandler.java:94]: Looks `yaml` to me. Can we get a comment here on why we do raw `println`s here?

**Review by donat — APPROVED** [review]
LGTM. I just have one minor request.

**Comment by donat** [comment]
❌ Wait, why is there an empty keyring file?

**Comment by big-guy** [comment]
@donat I thought the same thing, but it's not.  It's just the way the diff is shown (check out the branch and the file isn't empty)

File: subprojects/platform-native/src/main/java/org/gradle/nativeplatform/toolchain/internal/swift/SwiftDepsHandler.java (hunk 1 above)
Comment (JLLeitschuh) [reply subprojects/platform-native/src/main/java/org/gradle/nativeplatform/toolchain/internal/swift/SwiftDepsHandler.java:94]: @big-guy and I couldn't get the YAML generator to spit out the right file output

**Pull request merged**